target/linux/ramips/image/mt7621.mk

```diff
@@ -915,6 +982,25 @@ define Device/d-team_pbr-m1
 endef
 TARGET_DEVICES += d-team_pbr-m1
 
+define Device/dna_valokuitu-plus-ex400
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 117m
+  PAGESIZE := 2048
+  MKUBIFS_OPTS := --min-io-size=$$(PAGESIZE) --leb-size=124KiB --max-leb-cnt=96 \
+  		  --log-lebs=2 --space-fixup --squash-uids
+  DEVICE_VENDOR := DNA
+  DEVICE_MODEL := Valokuitu Plus EX400
+  IMAGES := factory.ubifs sysupgrade.tar
+  IMAGE/factory.ubifs := kernel-initramfs-bin | append-dtb | lzma | uImage lzma | \
```

Comment:
I think, we can create the whole *.y3 image here that can be flashed by the web-GUI and via U-Boot via HTTP and TFTP. Code for generation of Y3-image can be found here: https://dev.iopsys.eu/feed/targets/-/blob/7dbba8773155c0d6507bfce7a53926dfe418369e/iopsys-ramips/image/mt7621.mk#L122-182

Reply:
ah, my mistake. you are actually doing that. It was just the filename that confused me. Maybe factory.y3 is a more fitting name given that this is what the vendor-firmware uses as file extension?

Reply:
Replying to (somehow Github does not put replies via e-mail in the right thread :-( ):

> Is that the convention? Others I checked are '.img' or '.bin'. I though
> to use '.ubifs' because that's what it is. No strong opinion here.

Addmittely I did not know what the convention was. The ubifs just confused me, thought it was a UBIFS image. 
It seems like `factory.bin` is the convention.

```
❯ git grep 'IMAGE/factory'|wc -l
325
❯ git grep 'IMAGE/factory\.bin'|wc -l
224
❯ git grep 'IMAGE/factory\.img'|wc -l
41
```